styles.css

```diff
@@ -0,0 +1,71 @@
+article {
+	border: 1px solid lightblue;
+	width: 30%;
+	display: inline-block;
+	margin-bottom: 20px;
+	font-family: Arial, Helvetica, sans-serif;
+}
+
+img{
+	padding-left: 3.25em;
+	padding-top: 10px;
+	padding-bottom: 10px;
+	/*padding-right: 4em;*/
+	/*justify-content: center;*/
+}
+
+/*section img{
```

Comment:
if you decided not to use code delete it - you can always look at previous commits if you think you need it again